src/olympia/devhub/tasks.py

```diff
@@ -241,7 +241,7 @@ def validate_file_path(path, channel):
         # search plugins are validated directly by addons-server
         # so that we don't have to call the linter or validator
         results = deepcopy(amo.VALIDATOR_SKELETON_RESULTS)
-        annotations.annotate_search_plugin_validation(
+        annotations.annotate_search_plugin_restriction(
```

Comment:
Habit I think (of making sure not to touch too many places at once, especially for deprecations like that - and to be more dynamic regarding handling error messages etc), I'd leave it like that and will file a follow-up issue for a cleanup once it has settled in on prod to make sure .xml files are removed from `VALID_ADDON_FILE_EXTENSIONS` so that it'll go the same path as other format deprecations.

Reply:
I essentially went to do the same thing as with the thunderbird and SeaMonkey deprecation / removal to implement a separate annotation.